### PR TITLE
 [stable/redis] Refactor image properties in values.yaml

### PR DESCRIPTION
**Pull request by tompizmor**
From now it is possible to specify registry, repository, and tag for the docker image.

### CODE REVIEW

File: stable/redis/Chart.yaml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 name: redis
-version: 1.1.24
+version: 1.1.25
```
Comment (timstoop) [inline stable/redis/Chart.yaml]: As you're adding a new feature, you should bump the minor version.

**Review by timstoop — CHANGES_REQUESTED** [review]
Also, you might want to take a look at the e2e test result.

**Comment by carrodher** [comment]
@tompizmor @prydonius This PR was merged before the semver discussion, how should we proceed?

**Pull request merged**